analyses/cell-type-wilms-tumor-14/exploratory_analysis/03_cnv/README.md

```diff
@@ -0,0 +1,30 @@
+## Exploratory analysis
+
+#### CNV analysis
+The goal of this exploratory CNV analysis is to see if `CopyKat` and `inferCNV` can be useful methods for predicting tumor cells based on ploidy. This analysis uses the library `SCPCL000850` (sample `SCPCS000518`) to explore these methods.
+
+See below for details on how to run this exploratory CNV analysis.
+```bash
+cd /path/to/OpenScPCA-analysis
+cd analyses/cell-type-wilms-tumor-14/exploratory_analysis/03_cnv
+# required for rendering Rmd files
+export RSTUDIO_PANDOC="/usr/lib/rstudio/resources/app/bin/quarto/bin/tools"
+
+## CopyKat
+# generate CNV results using copykat
+Rscript ./03_runCopyKat.R
+# render Rmd
+Rscript -e "rmarkdown::render('./03_copykat_SCPCL000850.Rmd')"
+
+## inferCNV
+# generate inferCNV inputs
+step_name="03_cnv"
+scratch_dir_step="../../scratch/${step_name}/infercnv" && mkdir -p ${scratch_dir_step}
+Rscript ../../../cell-type-ewings/scripts/cnv-workflow/00-make-gene-order-file.R \
+  --local_ref_dir ${scratch_dir_step} \
+  --scratch_dir ${scratch_dir_step}
```

Comment:
It seems this script has a dependency you will need to add to your renv environment: `rtracklayer`. Since this package is not specifically used in _your_ module, however, you need to tell your module about it. There is a file in your module `components/dependencies.R` which we use for this reason - you can add the line `library(rtracklayer)` to this file as a way to tell renv that this package is needed in the environment. Then, you'll want to run:
```
renv::install("rtracklayer")
renv::snapshot() # this will only work properly if you have added library(rtracklayer) to dependencies.R already
```